### ISSUE TITLE
Cannot resolve 'file' or 'directory' ./Resolver

### ISSUE THREAD

**Issue by srph** [ericclemmons/react-resolver#28]
```
Module not found: Cannot resolve 'file' or 'directory' ./Resolver in <path/to/dist>
```

I just switched to Ubuntu Server 14.04. I just checked `/dist/`, and it doesn't seem to have the `Resolver.js` file, but only `resolver.js`

**Comment by ericclemmons** [comment]
WTF!  Stupid case sensitivity!

```
tree node_modules/react-resolver/
node_modules/react-resolver/
├── CHANGELOG.md
├── LICENSE
├── README.md
├── dist
│   ├── Container.js
│   ├── Context.js
│   ├── ResolverError.js
│   ├── index.js
│   ├── mixin.js
│   ├── resolver.js
│   └── utils.js
└── package.json
```

Fixing...

**Comment by srph** [comment]
Yay, thanks. If you're busy, I can send a PR.

**Comment by ericclemmons** [comment]
Just connected to internet & saw your offer.  I should've taken you up on it!

**Comment by ericclemmons** [comment]
Fixed with v1.1.3!

Thanks for letting me know :)

```
  tree node_modules/react-resolver/
node_modules/react-resolver/
├── CHANGELOG.md
├── LICENSE
├── README.md
├── dist
│   ├── Container.js
│   ├── Context.js
│   ├── Resolver.js
│   ├── ResolverError.js
│   ├── index.js
│   ├── mixin.js
│   └── utils.js
└── package.json
```

**Comment by srph** [comment]
Thanks!
